Assets/Scripts/UIManager.cs

```diff
@@ -7,11 +7,25 @@ public class UIManager : MonoBehaviour
 {
     [SerializeField]
     private Text _winText;
+    [SerializeField]
```

Comment:
preference for spaces between variable and the next `[SerializedField]` attribute